### PR TITLE
Tk 320 updates

### PR DESCRIPTION
**Pull request by tedk13**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,17 +6,20 @@
...
+    "build:icons": "cp node_modules/@ritterim/platform-ui/src/platform-icons.json src/data/icons.json"
   },
   "keywords": [],
   "author": "RIMdev",
   "license": "ISC",
   "dependencies": {
-    "@astrojs/mdx": "^0.12.0",
-    "astro": "^2.0.5"
+    "@astrojs/mdx": "^0.19.7",
+    "@ritterim/platform-icons": "^2.1.7",
```
[the remark this answers is not in the document]
Comment (tedk13) [reply package.json:17]: I did this to get the icon.json. _that_ did not shit with pui.

File: src/components/icons.astro [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      <span class="flex text--size-2xl text--base p-3">
+        <i class={`pi-${icon[0]}`}></i>
+      </span>
+      <span class="flex--center-content">pi-{icon[0]}</span>
+      <div class="button mt-2 tooltip-bottom" data-copy-button="" data-clipboard-text={`<i class="pi-${icon[0]}"></i>`}>Copy <i class="pi-clipboard"></i></div>
+    </li>
+  )}
+</ul>
+  
+</div>
```
[the remark this answers is not in the document]
Comment (tedk13) [reply src/components/icons.astro:18]: And _that's_ how you Astro

File: src/sass/_platform-ui-plugins.scss [hunk 3]
```diff
@@ -0,0 +1,7 @@
+.form .input-icons [class^=pi-] {
+  display: flex;
+}
+
+.toggle input[type=checkbox]+label [class^=pi-], .toggle input[type=radio]+label [class^=pi-] {
+  display: flex;
+}
```
[the remark this answers is not in the document]
Comment (tedk13) [reply src/sass/_platform-ui-plugins.scss:7]: Oh this wasn't the toggle. It was the checkboxes and radios. 
https://platformui.com/docs/forms#radio
Screen Shot 2023-07-17 at 1 02 03 PM

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (hougasian) [reply package.json:17]: we ship the `src` directory, in fact, you're copying it from PUI in your `cp` call above.

[the remark this answers is not in the document]
Comment (hougasian) [reply package.json:17]: you cam also up the PUI version here 😄

[the remark this answers is not in the document]
Comment (tedk13) [reply package.json:17]: We don't pull in pui as a package. I bet that's why I only did icons as a package here. Just to grab the json.

File: src/sass/_platform-ui-plugins.scss (hunk 3 above)
[the remark this answers is not in the document]
Comment (hougasian) [reply src/sass/_platform-ui-plugins.scss:7]: same answer though?

[the remark this answers is not in the document]
Comment (tedk13) [reply src/sass/_platform-ui-plugins.scss:7]: yeah, idk, the icons looked jacked up in the examples.

**Comment by hougasian** [comment]
Screenshot 2023-07-18 at 2 51 45 PM
`gtmID` isn't resolving 😢

**Review by hougasian — APPROVED** [review]

**Pull request merged**